test/WebJobs.Extensions.CosmosDB.Tests/Trigger/CosmosDBTriggerAttributeBindingProviderTests.cs

```diff
@@ -354,11 +354,14 @@ public async Task ValidChangeFeedOptions_Succeed(ParameterInfo parameter)
 
             CosmosDBTriggerBinding binding = (CosmosDBTriggerBinding)await provider.TryCreateAsync(new TriggerBindingProviderContext(parameter, CancellationToken.None));
 
+            CosmosDBTriggerAttribute cosmosDBTriggerAttribute = parameter.GetCustomAttribute<CosmosDBTriggerAttribute>(inherit: false);
+
             Assert.Equal(typeof(IReadOnlyList<Document>), binding.TriggerValueType);
             Assert.Equal(new Uri("https://fromSettings"), binding.DocumentCollectionLocation.Uri);
             Assert.Equal(new Uri("https://fromSettings"), binding.LeaseCollectionLocation.Uri);
-            Assert.Equal(10, binding.ChangeFeedProcessorOptions.MaxItemCount);
-            Assert.True(binding.ChangeFeedProcessorOptions.StartFromBeginning);
```

Comment:
When is the StartFromBeginning true now validated?

Reply:
its on line 363 where input (true) is being compared.